packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html

```diff
@@ -0,0 +1,98 @@
+            </div>
+        </oui-step-form>
+        <oui-step-form
+            header="{{:: 'domain_configuration_enable_web_hosting_dns_configuration' | translate }}"
+            on-submit="$ctrl.updateConfiguration()"
+        >
+            <p
+                data-translate="domain_configuration_web_hosting_dns_info_NO_CHANGE"
+            ></p>
+
+            <oui-checkbox model="$ctrl.dnsConfiguration.A">
+                <span
+                    data-translate="domain_configuration_enable_web_hosting_dns_configuration_A"
+                ></span>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    oui-popover="{{:: 'domain_configuration_web_hosting_dns_info_A' | translate}}"
+                ></button>
+            </oui-checkbox>
+            <oui-checkbox model="$ctrl.dnsConfiguration.MX">
+                <span
+                    data-translate="domain_configuration_enable_web_hosting_dns_configuration_MX"
+                ></span>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    oui-popover="{{:: 'domain_configuration_web_hosting_dns_info_MX' | translate}}"
+                ></button>
+            </oui-checkbox>
+        </oui-step-form>
+        <oui-step-form
+            header="{{:: 'domain_configuration_enable_web_hosting_offer_price' | translate }}"
+        >
+            <div class="row">
+                <oui-select-picker
+                    class="col-md-3 mb-3"
+                    ng-repeat="price in $ctrl.pricings track by price.duration"
+                    name="price"
+                    data-model="$ctrl.price"
+                    data-label="{{:: ('domain_configuration_enable_web_hosting_offer_START_' + price.duration) | translate }}"
+                    data-values="[price]"
+                >
+                    <oui-select-picker-label>
+                        <span
+                            data-translate="{{:: 'domain_configuration_enable_web_hosting_offer_START_' + price.duration }}"
+                        ></span>
+                        <ovh-manager-catalog-price
+                            price="price.value"
+                            user="$ctrl.user"
+                        >
+                        </ovh-manager-catalog-price>
+                    </oui-select-picker-label>
+                </oui-select-picker>
+            </div>
+        </oui-step-form>
+        <oui-step-form
+            header="{{:: 'domain_configuration_enable_web_hosting_activation' | translate }}"
+            on-focus="$ctrl.loadCheckout()"
+            loading="$ctrl.isLoadingCheckout"
+            valid="$ctrl.contractsValidated"
+            submit-text="{{:: 'domain_configuration_enable_web_hosting_activate' | translate }}"
+            on-cancel="$ctrl.goBack()"
+        >
+            <ovh-contracts-summary
```

Comment:
no payment method selection? how is the payment made?

Reply:
Order is free so we autopay the purchase